### PR TITLE
Standardize on sentence case

### PR DESCRIPTION
**Pull request by madnicendio**
Add fixes to both titles as well as for log messages, exceptions and errors.

### CODE REVIEW

File: common/rdr/RandomStream.cxx [hunk 1]
```diff
@@ -69,7 +69,7 @@ RandomStream::RandomStream()
   {
 #endif
 #endif
-    vlog.error("no OS supplied random source - using rand()");
+    vlog.error("No OS supplied random source: Using rand()");
```
Comment (CendioOssman) [inline common/rdr/RandomStream.cxx:72]: A comma suggests an explanation or more details. Perhaps a comma if the dash is undesirable?

File: vncviewer/Viewport.cxx [hunk 2]
```diff
@@ -260,7 +260,7 @@ void Viewport::setCursor(int width, int height, const Point& hotspot,
     if (data[i*4 + 3] != 0) break;
 
   if ((i == width*height) && dotWhenNoCursor) {
-    vlog.debug("cursor is empty - using dot");
+    vlog.debug("Cursor is empty: Using dot");
```
Comment (CendioOssman) [inline vncviewer/Viewport.cxx:263]: Same here.

File: win/rfb_win32/CleanDesktop.cxx [hunk 3]
```diff
@@ -191,7 +191,7 @@ void CleanDesktop::enableWallpaper() {
     ImpersonateCurrentUser icu;
 
     if (restoreActiveDesktop) {
-      vlog.debug("restore Active Desktop");
+      vlog.debug("Restore active desktop");
```
Comment (CendioOssman) [inline win/rfb_win32/CleanDesktop.cxx:194]: Dito

**Review by CendioOssman — CHANGES_REQUESTED** [review]
Looks mostly good. Nice work!

But I am lacking a rationale. Standardising on sentence case is fine by me, but it's good to have something for future questions.

Ideally something in the commit message(s), but at least here in the PR if it's too large.

File: common/rdr/RandomStream.cxx (hunk 1 above)
Comment (CendioOssman) [reply common/rdr/RandomStream.cxx]: A stray `r` sneaked in here instead. Otherwise, it looks good. :)

Comment (madnicendio) [reply common/rdr/RandomStream.cxx]: Nice catch, fixed now :)

**Pull request merged**